### PR TITLE
Link: Document options 'http' and 'https' to link_assume_external_targets

### PR DESCRIPTION
**Pull request by francoisfreitag**
tinymce/tinymce#5016 which fixes issue tinymce/tinymce#4335

### CODE REVIEW

**Comment by TheSpyder** [comment]
`TINY-` numbers are actually a different reference system :)

https://github.com/tinymce/tinymce/pull/5016 which fixes issue https://github.com/tinymce/tinymce/issues/4335

File: plugins/link.md [hunk 1]
```diff
@@ -113,6 +113,31 @@ tinymce.init({
 });
 ```
 
+### `link_force_external_targets`
+
+This option allows you to set whether TinyMCE should prepend a `http://` or `https://` prefix if the supplied link does not already contain a protocol prefix.
```
Comment (TheSpyder) [inline plugins/link.md]: Perhaps a short note about how this interacts with `link_assume_external_targets`?

**Comment by francoisfreitag** [comment]
Changes:

- Remove `link_force_external_targets` and add options `'http'` and `'https'` to existing `link_assume_external_targets`.

Ready for review.

**Comment by TheSpyder** [comment]
Approved with additional changes.

**Pull request merged**